tests/test_plotting.py

```diff
@@ -425,14 +446,12 @@ def test_show_axes():
     # if not closed correctly, a seg fault occurs when exitting
     plotter = pyvista.Plotter(off_screen=False)
     plotter.show_axes()
-    plotter.close()
```

Comment:
In theory these can all be deleted now that the `autouse` fixture calls `pyvista.close_all`. I can add that to this PR if desired.